moped-editor/src/views/projects/projectView/ProjectTimeline.js

```diff
@@ -507,17 +507,17 @@ const ProjectTimeline = ({ refetch: refetchSummary }) => {
                     updateExistingPhases(newPhaseObject);
 
                     const newPhase = newPhaseObject?.phase_name;
-                    const statusMapped = getStatusByName(newPhase);
```

Comment:
we were passing in the new phase, but the function was referring to it as a status. I found that confusing.